localstack/services/cloudformation/cloudformation_starter.py

```diff
@@ -55,6 +56,17 @@ def start_cloudformation(port=None, asynchronous=False, update_listener=None):
         return thread
 
 
+def set_moto_account_ids(resource_json):
+    def fix_ids(obj):
+        if isinstance(obj, dict):
+            for key, value in obj.items():
+                if 'arn' in key.lower() and isinstance(value, six.string_types):
+                    obj[key] = value.replace(TEST_AWS_ACCOUNT_ID, MOTO_ACCOUNT_ID)
```

Comment:
Is the intent to replace the test AWS account ID with the one used by moto?  Is there anywhere that the opposite is done, so that when moto uses its own, on response back to the caller it is returned with the test AWS account ID expected?